### DISCUSSION TITLE
Welcome to quickfixj Discussions!

### DISCUSSION THREAD

**Discussion by chrjohn** [quickfix-j/quickfixj#341] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JThoennes** [comment]
Hi Christoph and happy new year,
hope you are doing well and read this message.
I wonder why Maven shows me QF/J 2.2.1 but I cannot find neither a git tag nor release notes for this version.
Is this still in progress?
Best regards, Jörg Thönnes

**Reply by chrjohn** [reply]
Hi Jörg,
happy new year! Thanks, I am fine, hope you are fine too.
Hmm, where does Maven show you version 2.2.1? The version in master is 2.2.1-SNAPSHOT.
https://github.com/quickfix-j/quickfixj/blob/09d6104889ab39df19ef7a2ae52c3bdb65a3a070/pom.xml#L6

The last officially released version is 2.2.0 (https://github.com/quickfix-j/quickfixj/releases), 2.2.1 is still in the works.

Did you maybe confuse it with 2.1.1?
![Screenshot from 2021-01-19 00-00-12](https://user-images.githubusercontent.com/6644028/104969643-5dbf4400-59e9-11eb-9afc-9af6af7a2983.png)


Cheers,
Chris.

**Reply by JThoennes** [reply]
Hi Christoph,
for some reason my Maven offered me `...message-all` in version `2.2.1`. But the build did not really work, so do not mind.
Is this here the new preferred way to communicate with you or the QF/J project?
Cheers, Jörg

**Reply by chrjohn** [reply]
Hi Jörg,

>  Is this here the new preferred way to communicate with you or the QF/J project?

actually I do not care whether it is via mailing list or here. But on github it is probably more handy when referring to code, issues or PRs.

Cheers,
Chris.

**Comment by ansonye0302** [comment]
Hi Christoph,

Hope you are doing well.
I would like to see if multiple emails is supported in FIX message for tag 523 -> 803(8) please?

For example, if the submitter has 2 emails need to be notified, can we have something like below: 
A> 523=a@gmail.com;b@gmail.com|803=8 
The emails are separated by ; or should another tag 803 (8) need to be created? Or only one email is allowed at a time?
B> 523=a@gmail.com|803=8|523=b@gmail.com|803=8 

Thanks for reading this and looking forward to your reply.

Best Regards,
Anson

**Reply by chrjohn** [reply]
Hi Anson,

this is more a FIX Protocol question than a QuickFIX/J question. But I don't see any reason why multiple mail addresses should not be allowed. I would rather choose the duplicate PartySubID instead of using a separator.

Cheers,
Chris.

**Reply by ansonye0302** [reply]
Thanks Chris. I will try with duplicate PartySubID and see if the FIX message is valid.

**Reply by chrjohn** [reply]
The FIX message will be valid in any case. You can add as much repeating group entries as you want. The question is if it makes sense. But especially the case of multiple mail addresses sounds like a valid use case to me.

Here is some info from the FIX Trading Community forum (posted by Hanno Klein):

> ... the spec allows to have multiple entries for PartyRole or PartySubIDType of the same type as long as the semantics are clear. If there are multiple emails and a primary email address does not need to be specified, then it makes sense to have multiple entries of PartySubIDType(803)=8. It probably makes less sense to have PartySubIDType(803)=1 (Firm) more than once.
>
>FIX intentionally tries to be as generic as possible and leaves it up to the Rules of Engagement to add text for a given implementation. There it might say “Multiple entries of PartySubIDType(803)=8 are permitted and the first entry will be taken as primary email address.” Or it might say “Multiple entries of PartySubIDType(803)=8 are not permitted.”
>
>The spec is explicit where we feel it is necessary to enable/disable specific usages. Otherwise it is up to the (bilateral) agreement between counterparts.

If you have access to the forum, the link is: https://forum.fixtrading.org/t/specify-partyrole-452-3-client-id-twice-in-nopartyids-453/15651/5?u=christophjohn

**Reply by ansonye0302** [reply]
Thanks Chris for the detailed info. I will try with what you suggested. Thanks a lot.

**Comment by YA-IL** [comment]
Hi @chrjohn,
Is this community still active?

**Reply by chrjohn** [reply]
Hi @cypatorYA , of course. Best is to create a new discussion via https://github.com/quickfix-j/quickfixj/discussions/new/choose

**Reply by YA-IL** [reply]
thanks, @chrjohn I will open a Q&A discussion to [my question](https://github.com/quickfix-j/quickfixj/discussions/641), would really appreciate some help with my question, I could not find any clear answer to it also I published this question on [StackOverflow ](https://stackoverflow.com/questions/76404804/quick-fix-j-qfj-ssl-connection-between-initiator-and-acceptor-logic)a few days a go with no respond.

thanks
